app/src/pages/playground/PlaygroundDatasetExamplesTable.tsx

```diff
@@ -0,0 +1,280 @@
+import React, { ReactNode, useCallback, useMemo, useRef } from "react";
+import { graphql, useLazyLoadQuery, usePaginationFragment } from "react-relay";
+import {
+  CellContext,
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  Table,
+  useReactTable,
+} from "@tanstack/react-table";
+import { css } from "@emotion/react";
+
+import { Flex } from "@arizeai/components";
+
+import { AlphabeticIndexIcon } from "@phoenix/components/AlphabeticIndexIcon";
+import { JSONText } from "@phoenix/components/code/JSONText";
+import { borderedTableCSS, tableCSS } from "@phoenix/components/table/styles";
+import { TableEmpty } from "@phoenix/components/table/TableEmpty";
+import { usePlaygroundContext } from "@phoenix/contexts/PlaygroundContext";
+
+import type { PlaygroundDatasetExamplesTableFragment$key } from "./__generated__/PlaygroundDatasetExamplesTableFragment.graphql";
+import { PlaygroundDatasetExamplesTableQuery } from "./__generated__/PlaygroundDatasetExamplesTableQuery.graphql";
+import { PlaygroundDatasetExamplesTableRefetchQuery } from "./__generated__/PlaygroundDatasetExamplesTableRefetchQuery.graphql";
+
+function LargeTextWrap({ children }: { children: ReactNode }) {
+  return (
+    <div
+      css={css`
+        max-height: 300px;
+        overflow-y: auto;
+      `}
+    >
+      {children}
+    </div>
+  );
+}
+
+function JSONCell<TData extends object, TValue>({
+  getValue,
+}: CellContext<TData, TValue>) {
+  const value = getValue();
+  return (
+    <LargeTextWrap>
+      <JSONText json={value} space={2} />
+    </LargeTextWrap>
+  );
+}
+
+const PAGE_SIZE = 100;
+
+//un-memoized normal table body component - see memoized version below
```

Comment:
this is for column resizing and copied from the experiment compare table which is copied from here https://tanstack.com/table/latest/docs/framework/react/examples/column-resizing-performant